app/classes/Transvision/ShowResults.php

```diff
@@ -145,6 +145,21 @@ public static function getTranslationMemoryResults($strings, $search, $max_resul
     }
 
     /**
+     * Use the api to return a transliterated string
+     *
+     * @param string $string The string to be transliterated
+     * @param string $locale The target locale
+     *
+     * @return string The string transliterated into the target locale
+     */
+    public static function getTransliteratedString($string, $locale)
+    {
+        $request = new API(parse_url(API_ROOT . "transliterate/$locale/$string"));
+        $json = include MODELS . 'api/transliterate.php';
+
+        return $json[0];
```

Comment:
OK, my VM was missing php5-intl. Still, I think we should be a little more error proof.